ci/jobs/projects.yaml

```diff
@@ -32,14 +32,14 @@
 - project:
     name: pulp-dev
     os:
-        - f22
-        - f23
-        - rhel6
-        - rhel7
+        - 'f22'
+        - 'f23'
+        - 'rhel6'
+        - 'rhel7'
     pulp_version:
-        - 2.8
-        - 2.9
-        - 2.10:
+        - '2.8'
+        - '2.9'
+        - '2.10':
```

Comment:
These quotes make sense since yaml will likely try to cast "2.8" as a float, for example.